### ISSUE TITLE
Edit audio meta

### ISSUE THREAD

**Issue by GEkuL** [UnicornsOnLSD/finamp#209]
Hi there,

Could you add the feature to edit names, author, album, genre e.t.c?

I've seen this in Vanilla Music player - Vanilla Tag Editor. A pretty useful thing, I must say!

**Comment by UnicornsOnLSD** [comment]
That would be interesting, the only real customisation I have now is editing playlist names.

**Comment by GEkuL** [comment]
@UnicornsOnLSD yeah, that would be perfect for editing you media library in case of wrong artist, albums e.t.c.
